src/rgw/rgw_sal_motr.cc

```diff
@@ -176,6 +176,35 @@ inline std::string get_bucket_name(const std::string& tenant,  const std::string
     return bucket;
 }
 
+int static update_bucket_stats(const DoutPrefixProvider *dpp, MotrStore *store,
```

Comment:
Any reasons to keep this function static?

Reply:
sure will address with the next PR regarding Stats.